src/models/image-map.ts

```diff
@@ -51,8 +51,13 @@ export interface IImageHandlerStoreResult {
   displayUrl?: string;
   success: boolean;
 }
+interface IImageMap {
+  curriculumBaseUrl?: string;
+  unitCodeMap?: IMSTMap<ISimpleType<string>>;
+  getCurriculumUrl?: () => string | undefined;
```

Comment:
With the change below this could be:
```suggestion
  curriculumUrl?: string;
```

However it seems like I might not be understanding something.